papers/Raku_Steering_Council_Code.md

```diff
@@ -54,63 +46,92 @@ The Steering Council is a 7-person committee.
 The Steering Council shall work to:
 
 * Maintain the quality and stability of the Raku language, its
-compilers and its ecosystem.
-* Make contributing and learning it as accessible, inclusive, and sustainable as
-possible.
+compilers and its ecosystem;
+* Make contributing and learning the Raku language as accessible, inclusive, and sustainable as
+possible;
 * Formalize and maintain the relationship between the core team and
-The Raku Foundation<sup id="a1">[1](#f1)</sup>,
+The Raku Foundation<sup id="a1">[1](#f1)</sup>;
 * Establish appropriate decision-making processes for problem solving
-proposals,
+proposals;
 * Seek consensus among contributors and the core team before acting in
-a formal capacity,
+a formal capacity;
+* Promote the development of an inclusive, friendly and helpful community of Raku users;
+* Facilitate the promotion of a Code of Conduct by everyone in the Community, the Core Team and
+the Steering Council;
 * Act as a "court of final appeal" for decisions where all other
 methods have failed.
 
 
 ### Powers
 
-
 The Steering Council has broad authority to make decisions about the
-project.  For example, it can:
+development of the Raku language, the composition of the Core Team, and interactions with the 
+Raku users' community.  For example, it can:
 
 * Accept or reject problem-solving proposals
-* Enforce or update the project's code of conduct
+* Enforce the Code of Conduct
```

Comment:
I would like to keep this line unchanged.  The RSC will have the power to change a CoC.